components/FlexibleHeader/src/MDCFlexibleHeaderView.m

```diff
@@ -822,7 +822,15 @@ - (void)fhv_accumulatorDidChange {
 
 - (void)fhv_updateLayout {
   if (!_trackingScrollView) {
-    return;
+    // Even if we're not tracking a scroll view, our minimumHeight might have changed due to new
+    // safe area insets.
+    if (_minimumHeight > 0 && self.bounds.size.height != _minimumHeight) {
+      CGRect bounds = self.bounds;
+      bounds.size.height = _minimumHeight;
+      self.bounds = bounds;
+      [self fhv_commitAccumulatorToFrame];
+      return;
```

Comment:
We are moving this return call, should it be outside this if block?

Reply:
Good catch, done.